### ISSUE TITLE
image map / links not working

### ISSUE THREAD

**Issue by tentaurus** [thunderbird/thunderbird-android#3121]
There's an email with an image map (picture with links according to the x,y position).
But I can't open a link. No link is shown.
Works well with other email clients...

Is there a (new) setting for this?

Android 7.0
k-9 5.403

Matt

**Comment by philipwhiuk** [comment]
`map` isn't on JSoup's relaxed list :/

**Comment by tentaurus** [comment]
means what? Not supported? Wasn't it supported in fomer versions?

**Comment by philipwhiuk** [comment]
In 5.4 we changed our HTML sanitation solution from HtmlCleaner to JSoup. JSoup handles the cruft you get in email (see #2148 and similar issues) much better. Unfortunately even the relaxed white-list is pretty vicious and so we regressed various bits. We are re-adding this in bugfixes (and also testing to prevent future regressions).

Note that we need to sanitise content to prevent leaking of personal data etc.

I've created #3123 for this
